src/Core/MailTemplates/Handlebars/OrganizationUserInvitedWithSponsorship.text.hbs

```diff
@@ -0,0 +1,10 @@
+﻿{{#>BasicTextLayout}}
```

Comment:
I'm not sure what this template is doing, I think it doesn't belong.

Reply:
Resolved offline.